lib/reporter.js

```diff
@@ -161,7 +161,11 @@ var generateReport = function (options) {
         var screenShotDirectory;
         suite.reportAs = 'Features';
 
-        screenShotDirectory = options.output ? path.join(options.output, '..', 'screenshot') : 'screenshot/';
+        if(options.screenshotsDir) {
+            screenShotDirectory = options.output ? path.join(options.output, '..', options.screenshotsDir) : options.screenshotsDir;
```

Comment:
@anrodkin any reasons to use **_output_** as a parent directory since it's a default place? How about if it uses _parent_ directory to create a screenshot directory because we explicitly mention it as an option? 

**E.g.** the below options should store the screenshots to the `<parentDirectory>/screenshots` where as the report would be at `<parentDirectory>/report/*.html`

```
{
   ...
   ...
   output: /report/cucumber_report.html,
   screenshotsDirectory: /screenshots,
   storeScreenshots: true
}

```